src/libraries/System.Private.CoreLib/src/System/Span.cs

```diff
@@ -273,7 +266,7 @@ public bool MoveNext()
         /// Clears the contents of this span.
         /// </summary>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public void Clear()
+        public unsafe void Clear()
```

Comment:
Why is unsafe required here?  Are nint/nuint only allowed in unsafe contexts?

Reply:
`sizeof(nint)` is not a constant and therefore requires `unsafe` same as any other non constant sizeof.

Reply:
Ah, right.